pkg/blob/controllerserver.go

```diff
@@ -725,10 +726,18 @@ func (d *Driver) copyBlobContainer(_ context.Context, req *csi.CreateVolumeReque
 		return fmt.Errorf("srcContainerName(%s) or dstContainerName(%s) is empty", srcContainerName, dstContainerName)
 	}
 
-	klog.V(2).Infof("generate sas token for account(%s)", accountName)
-	accountSasToken, genErr := generateSASToken(accountName, accountKey, storageEndpointSuffix, d.sasTokenExpirationMinutes)
-	if genErr != nil {
-		return genErr
+	var accountSasToken string
+	if len(d.cloud.Config.AADClientSecret) > 0 || d.cloud.Config.AzureAuthConfig.UseManagedIdentityExtension {
```

Comment:
if `secrets` is not empty, then use sastoken mode; otherwise check sp or managed identity setting.

Reply:
check secrets before `copyVolume`, if it has accountSasToken then use it